joss/paper.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'climpred: Verification of weather and climate forecasts'
+tags:
+  - python
+  - climate
+  - forecasting
+  - geospatial
+  - big data
+authors:
+  - name: Riley X. Brady
+    orcid: 0000-0002-2309-8245
+    affiliation: 1
+  - name: Aaron Spring
+    orcid: 0000-0003-0216-2241
+    affiliation: 2
+affiliations:
+ - name: Department of Atmospheric and Oceanic Sciences and Institute of Arctic and Alpine Research, University of Colorado Boulder, Boulder, Colorado USA
+   index: 1
+ - name: International Max Planck Research School on Earth System Modelling and Max Planck Institute for Meteorology, Hamburg, Germany
+   index: 2
+date: 16 October 2020
+bibliography: paper.bib
+---
+
+<!-- 982 words total -->
+
+# Summary
+<!-- 71 words -->
+Predicting extreme events and variations in weather and climate provides crucial
+information for economic, social, and environmental decision-making. However,
+quantifying prediction skill for multi-dimensional geospatial model output is
+computationally expensive and a difficult coding challenge. The large datasets
+(order gigabytes to terabytes) require parallel and out-of-memory computing to be
+analyzed efficiently. Further, aligning the many forecast initializations with differing
+observational products is a straight-forward, but exhausting and error-prone exercise
+for researchers.
+
+<!-- 68 words -->
+To simplify and standardize forecast verification across scales from hourly weather to
+decadal climate forecasts, we built `climpred`: a community-driven python package for
+computationally efficient and methodologically consistent verification of ensemble
+prediction models. The code base is maintained through open-source development. It
+leverages `xarray` [@Hoyer:2017] to anticipate core prediction ensemble dimensions
+(ensemble `member`, `init`ialization date and `lead` time) and `dask` [@dask] to
```

Comment:
we could also link to https://conference.scipy.org/proceedings/scipy2015/matthew_rocklin.html

Reply:
I dont like the `team` rendering

Reply:
This is the suggested citation from the `dask` website. We can just do both.